src/responses/germany.ts

```diff
@@ -168,7 +173,12 @@ export async function GermanyCasesHistoryResponse(
 export async function GermanyWeekIncidenceHistoryResponse(
   days?: number
 ): Promise<GermanyHistoryData<{ weekIncidence: number; date: Date }>> {
-  if (days != null) {
```

Comment:
@Rubber1Duck Is it save to remove the null check? Because `0 === false`.

Reply:
the null check is still there! see line 181

Reply:
@marlon360 oh mann, sorry i now understand what you meen! will check this, but i think you are right!

Reply:
A request with parameter days = 0 with a check for zero (i.e. days !=0) is the response an empty array! The "0" as a possible parameter for days should therefore be excluded AND the null check must be done! will patch this soon!

Reply:
done